### PR TITLE
Hyt sensor driver

### PR DESCRIPTION
**Pull request by adkrprog**
## Description:
driver for Innovative Sensor Technology IST AG's HYT precision relative temperature/humidity sensor
https://www.ist-ag.com/sites/default/files/downloads/dhhyt271_d.pdf

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.3
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

File: tasmota/xsns_90_HYT.ino [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/*
+  xsns_90_HYT.ino - HYT I2C temperature and humidity sensor support for Tasmota
+
```
Comment (arendst) [inline tasmota/xsns_90_HYT.ino:3]: Change to unused id ``xsns_97_hyt.ino``

File: tasmota/xsns_90_HYT.ino [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#ifdef USE_I2C
+#ifdef USE_HYT
+/*********************************************************************************************\
+ * HYT  - Temperature and Humidity
+ *
+ * I2C Address: 0x28
+\*********************************************************************************************/
+
+#define XSNS_90              90
```
Comment (arendst) [inline tasmota/xsns_90_HYT.ino:30]: Change to ``XSNS_97     97``

File: tasmota/xsns_90_HYT.ino [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+#define XI2C_90              90 // See I2CDEVICES.md
```
Comment (arendst) [inline tasmota/xsns_90_HYT.ino:31]: Change to ``XI2C_68     68``

File: tasmota/xsns_90_HYT.ino [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  }
+}
+
+
+/*********************************************************************************************\
+ * Interface
+\*********************************************************************************************/
+
+bool Xsns90(uint8_t function)
+{
```
Comment (arendst) [inline tasmota/xsns_90_HYT.ino:109]: Change to ``Xsns97...``

File: tasmota/xsns_90_HYT.ino [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+  if (!I2cEnabled(XI2C_90)) { return false; }
```
Comment (arendst) [inline tasmota/xsns_90_HYT.ino:110]: Change to ``XI2C_68``

**Comment by arendst** [comment]
A bit expensive isn't it https://www.conrad.de/de/p/ist-sensor-feuchte-und-temperatur-sensor-1-st-hyt-271-messbereich-0-100-rf-l-x-b-x-h-10-2-x-5-1-x-1-8-mm-505675.html

**Comment by adkrprog** [comment]
> A bit expensive isn't it https://www.conrad.de/de/p/ist-sensor-feuchte-und-temperatur-sensor-1-st-hyt-271-messbereich-0-100-rf-l-x-b-x-h-10-2-x-5-1-x-1-8-mm-505675.html

Depending on the sales channel, the prices may differ. Do you need a sample?

**Comment by arendst** [comment]
Yes please. DM me at https://sidweb.nl/cms3/en/contact

**Comment by arendst** [comment]
Merged in a different way. Thx.

**Pull request closed without merging**